From cafa52081db07dcfcd67a7644f08ea54b75aa838 Mon Sep 17 00:00:00 2001
From: Zhenyi Zhou <iynehz@163.com>
Date: Thu, 31 Oct 2024 15:27:32 +0000
Subject: [PATCH] add 'realtek,rtl8201' to mdio compatible whitelist

---
 drivers/net/mdio/of_mdio.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/mdio/of_mdio.c b/drivers/net/mdio/of_mdio.c
index 7eb32ebb846d..1e71f3079f49 100644
--- a/drivers/net/mdio/of_mdio.c
+++ b/drivers/net/mdio/of_mdio.c
@@ -97,6 +97,7 @@ static const struct of_device_id whitelist_phys[] = {
 	{ .compatible = "marvell,88E1510", },
 	{ .compatible = "marvell,88E1514", },
 	{ .compatible = "moxa,moxart-rtl8201cp", },
+	{ .compatible = "realtek,rtl8201", },
 	{}
 };
 
